etcdutl/snapshot/v3_snapshot.go

```diff
@@ -175,8 +179,17 @@ func (s *v3Manager) Status(dbPath string) (ds Status, err error) {
 					if err != nil {
 						return fmt.Errorf("cannot unmarshal value, key: %q value: %q err: %w", k, v, err)
 					}
+					key := string(kv.Key)
+					// Version == 0 indicates logical deletion.
+					// refer to https://etcd.io/docs/v3.5/learning/data_model/
+					isActive := kv.Version > 0
+					if curr, exists := keyStates[key]; !exists || rev.Main > curr.rev {
+						keyStates[key] = struct {
+							rev    int64
+							active bool
+						}{rev.Main, isActive}
```

Comment:
- Pls use `isTombstone` to check whether it's a tombstone. We can expose the function. https://github.com/etcd-io/etcd/blob/c6b9e9e9277121b347f63eb82d28c2c79611557e/server/storage/mvcc/revision.go#L120
- No need to compare revision. They are guaranteed to be increasing.